docker-compose.kafka-setup.yml

```diff
@@ -0,0 +1,83 @@
+version: "3.2"
+services:
+    # zookeeper
+    zk1:
+        image: confluentinc/cp-zookeeper:3.3.0
```

Comment:
For a development environment there is apparently a Go-binary which is compatible with Kafka, it may be better? Most people who opt to use Kakfa will deploy it themselves or have it set up already.